trending_data/src/main/webapp/script.js

```diff
@@ -107,15 +107,17 @@ function isCountrySupportedbyYT(countryCode) {
  * @param {Marker} marker
  */
 function displayPosts(marker) {
-  if (!isCountrySupportedbyYT(marker.countryCode)) {
-    const ytErr = '<h2>Region not supported by YouTube</h2>';
-    windowsHandler.openwindow(marker, ytErr);
-  } else { // if country is supported, fetch data
-    fetch('/ListYTLinks?country-code=' + marker.countryCode).then((response) =>
-      response.json()).then((videos) => {
-      const vidNode = getVideosNode(videos);
-      windowsHandler.openwindow(marker, vidNode);
-    });
+  if (marker.countryCode != windowsHandler.getCountryCode()) {
+    if (!isCountrySupportedbyYT(marker.countryCode)) {
+      const ytErr = '<h2>Region not supported by YouTube</h2>';
+      windowsHandler.openwindow(marker, ytErr);
+    } else { // if country is supported, fetch data
+      fetch('/ListYTLinks?country-code=' + marker.countryCode).then((response) =>
```

Comment:
This now needs to be `'/GetTrendingYTVideos'` since the url has been changed in the `ListYTLinksServlet`.
![image](https://user-images.githubusercontent.com/41055600/91277252-3d1cf600-e78b-11ea-9cab-a87fc2fb7677.png)